recipe-server/client/control_new/state/requests/selectors.js

```diff
@@ -12,3 +13,17 @@ export function isRequestInProgress(state, id) {
   const request = getRequest(state, id);
   return request.get('inProgress');
 }
+
+
+export function areAnyRequestsInProgress(state) {
+  if (state.app.requests.size === 0) {
+    return false;
+  } else if (state.app.requests.size === 1) {
+    return state.app.requests.first().get('inProgress');
```

Comment:
This can become `...get('inProgress', false);` to prevent the `|| false` business later on.